docs/docsource/operation/XDM_GUI/file_operations/delete_files.rst

```diff
@@ -0,0 +1,88 @@
+.. _Delete Files:
+
+Delete Files
+============
+
```

Comment:
Maybe you could add a title in here "Delete non-versioned files" ?

Reply:
I'm not feeling this. The first task is to delete files, which you can do to both versioned and non-versioned files (It deletes a non-versioned file or the most recent version of a versioned file. If you're looking at it as an end user and don't have access to the versions, it's deleted from today going forward, and you may not know about its past). The second task is a case of deleting a specific type of file. Breaking it into versioned vs. non-versioned files to my mind presents a false dichotomy.

Reply:
Gotcha! Guess I didn't understand the second part properly 😉

Reply:
I speaks well of you that you can find the edge cases to dive right dive into.